src/assets/icons/index.js

```diff
@@ -0,0 +1,137 @@
+import { ReactComponent as Archive } from './archive.svg'
```

Comment:
We should scale this list down to only the icons needed by the toolbar and delete the unused svg files